WordPress/src/main/java/org/wordpress/android/util/HelpshiftHelper.java

```diff
@@ -179,13 +183,28 @@ public void registerDeviceToken(Context context, String regId) {
     }
 
     public void setTags(Tag[] tags) {
-        mMetadata.put(Support.TagsKey, Tag.toString(tags));
+        setTags(Tag.toString(tags));
+    }
+
+    public void setTags(String[] tags) {
+        mMetadata.put(Support.TagsKey, tags);
     }
 
     public void addTags(Tag[] tags) {
+        addTags(Tag.toString(tags));
+    }
+
+    public void addTags(String[] tags) {
         String[] oldTags = (String[]) mMetadata.get(Support.TagsKey);
         // Concatenate arrays
-        mMetadata.put(Support.TagsKey, ArrayUtils.addAll(oldTags, Tag.toString(tags)));
+        mMetadata.put(Support.TagsKey, ArrayUtils.addAll(oldTags, tags));
+    }
+
+    public void addPlanTags() {
+        Set<String> planTags = BlogUtils.planTags();
+        if (planTags != null) {
```

Comment:
It seems that `planTags()` will always return a non-null set (and it will be quite explicit if you choose to add the relevant return type annotation) so, feel free to remove this null-check. Thanks!